src/io_device_traverse.rs

```diff
@@ -22,6 +23,33 @@ pub enum ChannelTraverseStatus {
     Cancel,
 }
 
+#[derive(Debug, Eq, PartialEq)]
+pub enum TraverseDeviceDrEvent {
```

Comment:
I think this should be a generic type in the api because this type probably belongs in the mayastor repo and not here.

Reply:
Yes, sure. I'll use and enhance the existing enum DrEvent from io-engine, and make parameter generic in spdk-rs.

Reply:
modified to use generics.